ilastik/workflows/pixelClassification/pixelClassificationWorkflow.py

```diff
@@ -72,6 +73,9 @@ def __init__(self, shell, headless, workflow_cmdline_args, project_creation_args
         parser.add_argument('--random-label-value', help="The label value to use injecting random labels", default=1, type=int)
         parser.add_argument('--random-label-count', help="The number of random labels to inject via --generate-random-labels", default=2000, type=int)
         parser.add_argument('--retrain', help="Re-train the classifier based on labels stored in project file, and re-save.", action="store_true")
+        parser.add_argument('--trees-number', help='Number of trees for Vigra RF classifier.', type=int)
```

Comment:
I know I'm changing my mind here, but I think a better name for this argument would be `--tree-count`, which is more consistent with the terminology in the [`vigra.learning.RandomForest`](http://ukoethe.github.io/vigra/doc-release/vigranumpy/index.html#vigra.learning.RandomForest) constructor.